tests/components/cast/fixtures/bbc_radio_fourfm.m3u8

```diff
@@ -0,0 +1,4 @@
+#EXTM3U
+#EXT-X-VERSION:3
+#EXT-X-STREAM-INF:PROGRAM-ID=1,BANDWIDTH=101760,CODECS="mp4a.40.5"
+http://as-hls-ww-live.akamaized.net/pool_904/live/ww/bbc_radio_fourfm/bbc_radio_fourfm.isml/bbc_radio_fourfm-audio%3d96000.norewind.m3u8
```

Comment:
This points at another playlist?

Reply:
That's right, this is an hls master playlist, which refers to media playlists.